src-ui/lib/components/BooksTable.svelte

```diff
@@ -1,28 +1,36 @@
 <script lang="ts">
-  import type { Book, BookWithoutId } from '$lib/types/book.js'
+  import type { Book, NewBook } from '$lib/types/book.js'
```

Comment:
`NewBook` is not a more general partial Book for insertion -- and inherited from the drizzle schema.